pages/coinbase/signin.py

```diff
@@ -9,12 +9,12 @@
 class SigninPage:
 
     def __init__(self, driver, drivethrough=True, bypass_auth=True):
-
+        
         # sign in page title: Coinbase - Buy/Sell Digital Currency
-        if not drivethrough: driver.get(config.coinbase_domain + '/signin')
+        if drivethrough: driver.get(config.coinbase_domain + '/signin')     #did you throw in that not to trip me up?
```

Comment:
No this is not correct. revert.

I wrote documentation for this which should not be confusing. It's not merged in yet though. check the cleanup branch (it's in code review by the way).